app/AndroidManifest.xml

```diff
@@ -6,7 +6,9 @@
           android:versionName="2.26">
 
     <uses-sdk
-        tools:overrideLibrary="com.etsy.android.grid"
+        tools:overrideLibrary="com.etsy.android.grid, com.google.android.gms.measurement,
```

Comment:
My understanding is that it prevents the builds from failing, but puts the onus on not-importing or referencing those files onto us, so we have to be cautious to clip them out of incompatible files.

What's the Min SDK version for these?